jni/CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 0.8.0-wip
+
+- **Breaking Change** ([#394](https://github.com/dart-lang/jnigen/issues/394)):
```

Comment:
Leave a reference to https://dart.dev/language/error-handling here or in the issue description.